node/taskfiles/taskfile.local.yml

```diff
@@ -105,7 +105,7 @@ tasks:
   test-reporter:
     dotenv: [".env"]
     cmds:
-      - go test ./pkg/reporter -v
+      - go test ./pkg/reporter -v -count=1
```

Comment:
is -count=1 necessary here? 😅

Reply:
ahh i keep forgetting it since I need it locally 😢